metrics/prometheus.go

```diff
@@ -150,10 +150,16 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc) *PrometheusCo
 				},
 			}, {
 				name:        "container_cpu_usage_seconds_total",
-				help:        "Cumulative cpu time consumed per cpu in seconds.",
+				help:        "Cumulative cpu time consumed in seconds.",
```

Comment:
IIUC, users either get total OR per-cpu?  Why not just add a new metric for total?

Reply:
See previous comment.

Reply:
Ok, I agree with this.